src/main/java/mate/academy/springboot/datajpa/dto/mapper/ProductMapper.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.dto.ProductRequestDto;
+import mate.academy.springboot.datajpa.dto.ProductResponseDto;
+import mate.academy.springboot.datajpa.model.Product;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class ProductMapper implements DtoRequestMapper<ProductRequestDto, Product>,
+        DtoResponseMapper<ProductResponseDto, Product> {
+    private final CategoryService categoryService;
+
+    @Override
+    public Product fromDto(ProductRequestDto object) {
+        Product product = new Product();
```

Comment:
use model dto naming
```suggestion
    public Product fromDto(ProductRequestDto dto) {
        Product model = new Product();
```

Reply:
in other places like this too